client/App.jsx

```diff
@@ -1,17 +1,47 @@
-import React from 'react';
-
+import React, { Component } from 'react';
+import Login from './Login.jsx';
+import SignUp from './SignUp.jsx';
+import {
+  BrowserRouter as Router,
+  Routes,
+  Route,
+  Link,
+} from 'react-router-dom';
 
 import './styles/styles.css';
 
-const App = props => {
+class App extends Component {
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+
+    return (
```

Comment:
Let's use the <BrowserRouter> component here. It's better for keeping the UI in sync with the URL